cfgov/jinja2/v1/_includes/molecules/_vars-content.html

```diff
@@ -0,0 +1,9 @@
+{% macro _privacy_statement() %}
+<p class="short-desc">
```

Comment:
Should the `<p>` block be indented? I've been doing that but can switch back to inline with the macro wrapper.